docs/05-concepts/06-database/10-raw-access.md

```diff
@@ -1,45 +1,80 @@
 
-``` dart
-[
-  row {
-    table {
-      column: content,
-    },
-  },
-]
+```dart
+int result = await session.db.unsafeExecute(
+  r'DELETE FROM mytable WHERE id = @id',
+  parameters: QueryParameters.named({'id': 1}),
+);
 ```
 
-Above is a visual aid of the data structure.
+## `unsafeSimpleQuery`
+
+Similar to `unsafeQuery`, but uses the simple query protocol. This protocol does not support parameter binding, making it more susceptible to SQL injection. **Use with extreme caution and only when absolutely necessary.**
+
+Simple query mode is suitable for:
+
+* Queries containing multiple statements.
+* Situations where the extended query protocol is not available (e.g., replication mode or with proxies like PGBouncer).
+
 
 ```dart
-List<Map<String, Map<String, dynamic>>> result = await session.db.unsafeQueryMappedResults(
-  'SELECT * FROM mytable LEFT JOIN othertable ON mytable.id = othertable.mytableid',
-);
+  DatabaseResult result = await session.db.unsafeSimpleQuery(
+      r'SELECT * FROM mytable WHERE id = 1; SELECT * FROM othertable;'
+  );
 ```
 
-Use the `unsafeExecute` method when no result is needed. The return value represents the number of rows that changed.
+## `unsafeSimpleExecute`
+
+Similar to `unsafeExecute`, but uses the simple query protocol. It does not return any results. **Use with extreme caution and only when absolutely necessary.**
+
+Simple query mode is suitable for the same scenarios as `unsafeSimpleQuery`.
 
 ```dart
-int result = await session.db.unsafeExecute(
-  'DELETE FROM mytable WHERE id = 1',
+  int result = await session.db.unsafeSimpleExecute(
+      r'DELETE FROM mytable WHERE id = 1; DELETE FROM othertable;'
+  );
+```
+
+## Query Parameters
+
+To protect against SQL injection attacks, always use query parameters when passing values into raw SQL queries. The library provides two types of query parameters:
+
+* **Named Parameters:** Use `@` to denote named parameters in your query and pass a `Map` of parameter names and values.
+* **Positional Parameters:** Use `$1`, `$2`, etc., to denote positional parameters and pass a `List` of parameter values in the correct order.
+
+```dart
+// Named parameters
+var result = await db.unsafeQuery(
+  r'SELECT id FROM apparel WHERE color = @color AND size = @size',
+  QueryParameters.named({
+    'color': 'green',
+    'size': 'XL',
+  }));
+
+// Positional parameters
+var result = await db.unsafeQuery(
+  r'SELECT id FROM apparel WHERE color = $1 AND size = $2',
+  QueryParameters.positional(['green', 'XL']),
 );
 ```
 
-in the above example, the `result` variable will contain the number of rows that were deleted.
+> [!CAUTION]
+> Always sanitize your input when using raw query methods. For the `unsafeQuery` and `unsafeExecute` methods, use query parameters to prevent SQL injection. Avoid using `unsafeSimpleQuery` and `unsafeSimpleExecute` unless the simple query protocol is strictly required.
```

Comment:
Suggestion:

Keep the
```md
:::danger
content
:::
```

As it renders a nice visual box

<img width="979" alt="Screenshot 2024-05-20 at 13 12 22" src="https://github.com/serverpod/serverpod_docs/assets/7395515/4eeb9887-e24f-41df-a3de-1db6d027ad29">